all/pom.xml

```diff
@@ -35,12 +35,117 @@
 
   <profiles>
     <profile>
-      <id>full</id>
+      <id>uber</id>
+      <repositories>
+        <repository>
+          <id>staged-releases</id>
+          <name>Staged Releases</name>
+          <url>https://oss.sonatype.org/service/local/repositories/${stagingRepositoryId}/content/</url>
+        </repository>
+      </repositories>
+
+      <dependencies>
+        <!-- Depend on all our native jars -->
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>netty-transport-native-epoll</artifactId>
+          <version>${project.version}</version>
+          <classifier>linux-x86_64</classifier>
```

Comment:
I know these are the platforms we typically release from but should we use `${jni.classifier}` here in case we change that, or someone else wants to generate an artifact on some other system?